ui-react/packages/atlasmap-standalone/src/App.tsx

```diff
@@ -38,6 +39,24 @@ const App: React.FC = () => {
     [importAtlasFile]
   );
 
+  const [exportDialog, openExportDialog] = useSingleInputDialog({
+    title: 'Export Mappings and Documents.',
+    content: 'Please enter a name for your exported catalog file',
+    placeholder: 'atlasmap-mapping.adm',
+    onConfirm: (closeDialog, value) => {
+      closeDialog();
+      if (value === undefined) {
```

Comment:
I think this callback should guarantee not to be fired in case of no value. This could be achieved by setting the text input as required and do some validation on the minimum length. Doing this should streamline the code.